src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs

```diff
@@ -4235,5 +4235,20 @@ public override SyntaxNode TupleExpression(IEnumerable<SyntaxNode> arguments)
             => SyntaxFactory.TupleExpression(SyntaxFactory.SeparatedList(arguments.Select(AsArgument)));
 
         #endregion
+
+        #region Patterns
+
+        internal override bool SupportsPatterns(ParseOptions options)
```

Comment:
Should this method be in CSharpSyntaxFactsService.cs instead?  There's a similar one in there for SupportsThrowExpression.

Reply:
Sure.  i could do that.